### PR TITLE
Fix default type size with scale

### PR DESCRIPTION
**Pull request by kairichard**
This fixes issue #895

### CODE REVIEW

File: src/Propel/Generator/Platform/DefaultPlatform.php [hunk 1]
```diff
@@ -1400,7 +1400,7 @@ public function normalizeTable(Table $table)
 
         foreach ($table->getColumns() as $column) {
             if ($column->getSize() && $defaultSize = $this->getDefaultTypeSize($column->getType())) {
-                if (intval($column->getSize()) === $defaultSize) {
+                if (null === $column->getScale() && intval($column->getSize()) === $defaultSize) {
```
Comment (marcj) [inline src/Propel/Generator/Platform/DefaultPlatform.php]: Shouldn't this be a logical OR?

Comment (kairichard) [inline src/Propel/Generator/Platform/DefaultPlatform.php]: `getScale` can still return `null` while `getSize` returns the `defaultSize` and only then can the `size` be overriden to `null`

Comment (marcj) [inline src/Propel/Generator/Platform/DefaultPlatform.php]: aah,I see :)

**Comment by marcj** [comment]
Thanks @kairichard :)

**Pull request merged**